src/ptf/dataplane.py

```diff
@@ -27,13 +27,20 @@
 from threading import Thread
 from threading import Lock
 from threading import Condition
-import ptfutils
-import netutils
-import mask
+from . import ptfutils
+from . import netutils
+from . import mask
```

Comment:
`import .ptfutils` works as well and it is more readable in my view.

Reply:
If you want to keep compatibility with Python2 (as below), you need the following:
```
if sys.version_info.major == 2:
    import ptfutils
else:
    import .ptfutils```